src/panels/GettingStartedPanel/Steps/ButtonWrapper.tsx

```diff
@@ -0,0 +1,21 @@
+import { PropsWithChildren } from 'react';
+import { MantineStyleProps, Paper } from '@mantine/core';
+
+interface Props extends PropsWithChildren, MantineStyleProps {
+  withBorder?: boolean;
+}
+
+export function ButtonWrapper({ withBorder = false, children, ...styleProps }: Props) {
```

Comment:
Since this is a component with a very specific use case and a general name, maybe add a short comment above it explaining the intended use case?